executor/executor_test.go

```diff
@@ -2515,6 +2515,15 @@ func (s *testSuite) TestIssue5341(c *C) {
 	tk.MustQuery("select * from test.t where a < 1 order by a limit 0;").Check(testkit.Rows())
 }
 
+func (s *testSuite) TestContainDotColumn(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists test.t1")
+	tk.MustExec("create table test.t1(t1.a char)")
+	tk.MustExec("drop table if exists t2")
+	tk.MustExec("create table t2(a char, t2.b int)")
```

Comment:
add test for 
create table t(s.a char);

Reply:
Already add.